tests/test_processing_stack.py

```diff
@@ -0,0 +1,117 @@
+import logging
+import os
+import time
+
+from pytest import mark
+
+APPLICATION_TAG_KEY = "ApplicationName"
+APPLICATION_TAG_VAL = "geospatial-data-lake"
+
+ENVIRONMENT_TAG_KEY = "EnvironmentType"
+ENVIRONMENT_TAG_VAL = os.environ.get("DEPLOY_ENV", "dev")
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def detect_compute_environment(batch_client):
+    compute_environment_detection_response = batch_client.describe_compute_environments()
```

Comment:
When using any AWS API calls which paginate we should (as discussed) do something like `assert response["nextToken"] is None` to get an alert if we ever need to start supporting pagination even in this test code.

Reply:
Solved in 9f529447d3faf75bb7c0ae62f9b923ae4cbce5c1